### ISSUE TITLE
Suggestion for option: Automatically create a card relation tag

### ISSUE THREAD

**Issue by galantra** [Arthur-Milchior/anki-copy-note#8]
Hi Arthur,

I have an idea regarding two of your add-ons. See how you find it; maybe you want to implement it in the future.

Copied notes tend to be similar to each other. Therefore, users might want them to be treated like siblings, i.e. buried.
Fortunately, you've already created the [relation add-on](https://github.com/Arthur-Milchior/anki-relation) for this. The relation process still requires manual steps. I imagine it could be automated with an option in "copy notes" to enable it, e.g. "add relation tag to copy and original".
As you are the creator of both add-ons, I assume you are the best for ensuring compatibility.
One issue that's come to my mind is that people can have differently named tags in their "Bury related notes" config.

Furthermore, this will bring your add-on "Bury related notes" to greater popularity because "Copy notes" is already widely used.

**Comment by Arthur-Milchior** [comment]
As it's not hard to implement, I've done it. May I suggest tipping ?

Honestly, even if I didn't create "relation" add-on, it wouldn't have been hard to add this feature to the copy add-on. Relating simply consists in add-on a tag. 

While I do agree that the copy add-on is widely used (3662 downloads according to ankiweb), while "bury related note" is almost not used (94 downloads) may I ask how you knew those fact ?
By the way, I'm not looking for popularity for every add-on here. I mean; all of the tips I received since I started are still less than a day of wage. And it does not helps me have better add-on for my own uses, since it's quite rare that add-on authors collaborate for what I've seen up to today

**Comment by galantra** [comment]
You are right. I should really give some material appreciation for all the help you've provided. Especially since I'm one of the major beneficiaries of your contributions. I feel that gratitude is not conveyed sufficiently over the Internet in general. 
So I'm a patreon now. It's a small amount, but I'm going to maintain it as a long-term commitment. I've also set up a reminder to increase the amount when I have more financial leeway, i.e. graduated and a job. 

I too see that add-on developers don't often collaborate. I think it's partly because most add-ons aren't maintained after creation. With active developers, especially those who've made more add-ons, I see a higher chance.  
It's similar for deck collaboration. Although I think here collaboration makes less sense because decks are more effective when there well-understood and personalized.

As for the download numbers, I don't know them directly (no magic tricks, don't worry). I've inferred them from the count of reviews on ankiweb.
